packages/lib-classifier/src/store/Project/Project.js

```diff
@@ -26,7 +26,7 @@ const Project = types
     },
 
     get isVolumetricViewer() {
-      return self.experimental_tools.includes('volumetricViewer')
+      return self.experimental_tools.includes('volumetricProject')
```

Comment:
I noticed the label on your staging project's [admin page](https://www.zooniverse.org/admin/project_status/kieftrav/volumetric-viewer?env=staging) is still `VolumetricViewer`. Are there still updates needed in the PFE codebase to align it with `VolumetricProject`?